lib/auth/auth_with_roles_test.go

```diff
@@ -67,7 +66,7 @@ func TestGenerateUserCerts_MFAVerifiedFieldSet(t *testing.T) {
 	for _, test := range []struct {
 		desc           string
 		getMFAResponse func() *proto.MFAAuthenticateResponse
-		wantErr        bool
+		errMsg         string
```

Comment:
FYI, I tend to use `wantErr string` in tests like this, so the name was alright IMO. (No need to change it though.)